config/locales/en.yml

```diff
@@ -103,13 +178,170 @@ en:
+      business_description: Tools and guidance for businesses
+      childcare_parenting: Childcare and parenting
+      childcare_parenting_description: Includes giving birth, fostering, adopting,
+        benefits for children, childcare and schools
+      citizenship: Citizenship and living in the UK
+      citizenship_description: Voting, community participation, life in the UK, international
+        projects
+      disabilities: Disabled people
+      disabilities_description: Includes carers, your rights, benefits and the Equality
+        Act
+      driving: Driving and transport
+      driving_description: Includes vehicle tax, MOT and driving licences
+      education: Education and learning
+      education_description: Includes student loans, admissions and apprenticeships
+      employing_people: Employing people
+      employing_people_description: Includes pay, contracts and hiring
+      environment_countryside: Environment and countryside
+      environment_countryside_description: Includes flooding, recycling and wildlife
+      housing_local_services: Housing and local services
+      housing_local_services_description: Owning or renting and council services
+      justice: Crime, justice and the law
+      justice_description: Legal processes, courts and the police
+      tax: Money and tax
+      tax_description: Includes debt and Self Assessment
+      visas_immigration: Visas and immigration
+      visas_immigration_description: Apply to visit, work, study, settle or seek asylum
+        in the UK
+      working: Working, jobs and pensions
+      working_description: Includes holidays and finding a job
+    index:
+      brexit: 'Brexit: check what you need to do'
+      calculate_vehicle_tax: Vehicle tax rates
+      consultations: consultations
+      coronavirus: Coronavirus (COVID-19)
+      coronavirus_restrictions: Coronavirus restrictions remain in place across the
+        country.
+      council_tax_bands: Council Tax bands
+      covid: 'Coronavirus (COVID-19): rules'
+      departments_and_policy: Departments and&nbsp;policy
+      driving_test: Driving theory test
+      find_job: Find a job
+      get_tested: Get tested for COVID-19
+      get_tested_description: Get a test to check if you have coronavirus (COVID-19)
+        on NHS.UK. You do not need to have symptoms to get a test.
+      intro: The best place to find government services and information
+      intro_simpler: Simpler, clearer, faster
+      job_search: Find a job
+      jobseekers_allowance: Jobseeker's Allowance
+      merged_websites: The websites of all <a href="/government/organisations" class="govuk-link">government
+        departments</a> and many other agencies and public bodies have been merged
+        into GOV.UK.
+      ministerial_departments: Ministerial departments
+      more: More on GOV.UK
+      most_active: Most active
+      news: news and communications
+      other_agencies: Other agencies and public&nbsp;bodies
```

Comment:
Presume we can replace &nbsp; with a space

Reply:
I actually think this is intentionally, to ensure that `public bodies` is always kept together on the same line, rather than splitting the two words over two lines.

Reply:
Ah ok, thanks!

Reply:
I think it's safe to keep it also on the `Departments and&nbsp;policy` above